miluv/utils.py

```diff
@@ -4,8 +4,89 @@
 from scipy.spatial.transform import Rotation
 import scipy as sp
 import yaml
-import matplotlib.pyplot as plt
 
+from pymlg import SE3
+
+def get_anchors():
+    """
+    Get anchor positions.
+    
+    Returns: 
+    dict
+    - Anchor positions.
+    """
+    
+    with open(f"config/uwb/anchors.yaml", "r") as file:
+        anchor_positions = yaml.safe_load(file)
+    
+    for constellation in anchor_positions:
+        anchors = list(anchor_positions[constellation].keys())
+        for anchor in anchors:
+            pos = np.array([eval(anchor_positions[constellation][anchor])]).reshape(3, 1)
+            anchor_positions[constellation][int(anchor)] = pos
+            anchor_positions[constellation].pop(anchor)
+    
+    return anchor_positions
+
+def get_tag_moment_arms():
+    """
+    Get tag moment arms.
```

Comment:
Change from "Get tag moment arms." to "Get tag moment arms in the robot's body frame." so that readers get that information right from the start.